packages/app/src/pages/index.tsx

```diff
@@ -47,27 +50,73 @@ const HomePage: React.FC<Props> = () => {
         </div>
       );
       const direction = (
-        <article className="w-72 mx-auto">
+        <article className="lg:max-w-[400px] px-10 mx-auto">
           <h1 className="text-lg font-bold mb-4">{t('welcomeMessage')}</h1>
           <p className="mb-6 text-sm leading-relaxed">{t('description')}</p>
           <FilledButton.renewal
-            className="mb-21 w-full"
+            className="w-full"
             cs={COLOR_SYSTEM.PRIMARY}
             label={t('startButtonLabel')}
             onClick={handleDashboardButtonClick}
           />
-          <NavigationLinks.renewal
-            className="flex gap-2"
-            on={COLOR_SYSTEM.SURFACE_VARIANT}
-          />
-          <NavigationServices.renewal
-            className="mt-6 flex gap-2"
-            on={COLOR_SYSTEM.SURFACE_VARIANT}
-          />
-          <NavigationLanguages
-            className="mt-6 flex gap-2"
-            on={COLOR_SYSTEM.SURFACE_VARIANT}
-          />
+          <div className="text-xs mt-20 [&>ul]:flex [&>ul]:gap-x-4 [&>ul]:gap-y-2 [&>ul]:flex-wrap space-y-4">
```

Comment:
I too feel that arbitrary variants such as `[&>ul]` are useful.
However, I also think that they can become complicated if used too often.

In this case, it would be better to set a className for each `ul` element.

Reply:
How about style changing? It costs a lot.
Using a css modules is better ?

Reply:
In my opinion, it is more confusing in many cases to have the style applied to a new ul Element when it is added than to have all the styles changed.
I think all style changes are easy fixes, although they may be costly.

Reply:
Applying styles with no classes happens when not using tailwind. It's like the css cascading, and I think it makes sense.